docs/01-overview/main-areas/application-connectivity/ac-04-security.md

```diff
@@ -4,24 +4,9 @@ title: Security
 
 ## Client certificates
 
-To provide maximum security, Application Connector uses the TLS protocol with Client Authentication enabled. As a result, whoever wants to connect to Application Connector must present a valid client certificate, which is dedicated to a specific Application. In this way, the traffic is fully encrypted and the client has a valid identity.
+To provide maximum security, in the [Compass mode](./README.md), Application Connector uses the TLS protocol with Client Authentication enabled. As a result, whoever wants to connect to Application Connector must present a valid client certificate, which is dedicated to a specific Application. In this way, the traffic is fully encrypted and the client has a valid identity.
```

Comment:
We can add here few words that TLS Mutual Authentication is used to receive events from external system that Application Connector is subscribed for. 

For example: 
To provide maximum security, Application Connector uses the TLS protocol with Client Authentication enabled to receive events from external systems. As a result, any external system that wants to send events to Application Connector must present a valid client certificate generated when this system is registered in Compass.

Reply:
It would narrow the scope to just events. One of the reasons to connect an external solution is to send events, so it's kind of already included. I'll leave it as is.